ADJ-Internship/WebApp/Views/PurchaseOrder/Index.cshtml

```diff
@@ -0,0 +1,77 @@
+﻿@model ADJ.BusinessService.Dtos.OrderDTO;
+
+<h2>PO Management</h2>
+
+<p>
+    @Html.ActionLink("New PO", "Create")
```

Comment:
@Html is old style, research for ASP.NET Core syntax

Reply:
change from @html style to new ASP.NET Core syntax